.github/ISSUE_TEMPLATE/event.yml

```diff
@@ -0,0 +1,126 @@
+name: Create Event
+description: Submit a new event to be added to the community calendar.
+title: "[Event] "
+labels: ["event"]
+assignees: []
+
+body:
+  - type: input
+    id: title
+    attributes:
+      label: Title of the Event
+      placeholder: e.g. Django (birth)day Mombasa
+    validations:
+      required: true
+
+  - type: dropdown
+    id: event_type
+    attributes:
+      label: Event Type
+      options:
+        - online
+        - meetup
+        - conference
+    validations:
+      required: true
```

Comment:
Inspired by @Mitchina's schema, there are now two fields:

event_type: online, in_person
event_category: meetup, conference

I suppose we can supplement them later. But seems fine to split them up now already.